draft-ietf-rats-corim.md

```diff
@@ -1645,74 +1645,87 @@ all Endorsements in the group are silently ignored.
 Each group is processed independently of other groups. If a group fails to match
 the Accepted Claims Set then this does not affect the processing of other groups.
 
-#### Matching a Reference Value against the Accepted Claims Set {#sec-match-one-ref-val}
-
-This section describes how a Reference Value is matched against Evidence in the Accepted
-Claims Set.
-If any part of the processing indicates that the Reference Value does not match then the remaining steps in this section are skipped for that group.
-
-A Reference Value consists of an `environment-map` plus a `measurement-map`. In the
-`reference-triple-record` these are encoded together. In other triples multiple
-Reference Values are represented more compactly by letting one `environment-map`
-apply to multiple `measurement-map`s.
-
-The Verifier first looks for entries in the Accepted Claims Set with the same
-`environment-map` as the Reference Value. These are the candidate claims. If there are
-no candidate claims then the Reference Value does not match.
-
-A Verifier SHALL compare two `environment-map`s using a binary comparison of the CBOR
-encoded objects.
-
-A Verifier SHOULD convert `environment-map` into a form which meets CBOR Core
-Deterministic Encoding Requirements {{-cbor}} before performing the binary comparison.
-
-If the Reference Value contains an `authorized-by` field then the Verifier
-SHALL modify the candidate claims set to remove Claims whose `authorized-by`
-field does not contain one of the keys listed in the Reference Value
+#### Matching a stateful environment against the Accepted Claims Set {#sec-match-one-ref-val}
+
+[^issue]: There were two interpretations of the meaning of `Reference Value` and the adopted
+meaning does not match the version in the current text. I will submit a further PR to replace
+most uses of "Reference Value" in section 5 with "stateful environment".
+Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/178
```

Comment:
I think this will not render as you may expect :-)

```suggestion
> There were two interpretations of the meaning of `Reference Value` and the adopted meaning does not match the version in the current text. I will submit a further PR to replace most uses of "Reference Value" in section 5 with "stateful environment".  [tracked-at] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/178
```

Reply:
I expect to delete this paragraph in the next change, happy to remove it in this one if you prefer